deploy/complete/helm-chart/mushop/charts/nats/values.yaml

```diff
@@ -0,0 +1,14 @@
+replicaCount: 1
+
+image:
+  repository: nats
+  tag: 2.1.2
+  pullPolicy: IfNotPresent
+
+osb: {}
+
+service:
+  port:
```

Comment:
These ports need to be defined? They are used outside the service?

Reply:
Yes - NATS needs these ports but they are not used outside the service.  All nats services are used by other pods. there is nothing mapped to an ingress.